analyzer/src/main/kotlin/managers/Pip.kt

```diff
@@ -420,7 +422,10 @@ class Pip(
     }
 
     private fun getLicenseFromLicenseField(value: String?): String? {
-        if (value.isNullOrBlank() || value == "UNKNOWN" || value.lines().size > 1) return null
+        if (value.isNullOrBlank() || value == "UNKNOWN") return null
+
+        val isShortString = value.length <= SHORT_STRING_MAX_CHARS && value.lines().size == 1
```

Comment:
It would nice to have the documentation link from the commit message also here as a code comment, as reference why this check is done.